api/src/pcapi/core/offers/api.py

```diff
@@ -1389,3 +1390,82 @@ def get_stocks_stats(offer_id: int) -> offers_serialize.StocksStats:
             },
             status_code=404,
         )
+
+
+def fill_offer_extra_data_from_product_data(product_id: int) -> None:
+    offer_ids = []
+
+    product = models.Product.query.filter_by(id=product_id).one_or_none()
+
+    if not product:
+        raise exceptions.ProductNotFound()
+
+    try:
+        with transaction():
+            offers = models.Offer.query.filter_by(productId=product_id).all()
+
+            offer_ids = [offer.id for offer in offers]
+
+            for offer in offers:
+                offer.name = product.name
+                offer.extraData["gtl_id"] = product.extraData.get("gtl_id")
+                offer.extraData["csr_id"] = product.extraData.get("csr_id")
+                offer.extraData["code_clil"] = product.extraData.get("code_clil")
+                offer.extraData["isbn"] = product.extraData.get("isbn")
+                offer.extraData["dewey"] = product.extraData.get("dewey")
+                offer.extraData["rayon"] = product.extraData.get("rayon")
+                offer.extraData["author"] = product.extraData.get("author")
+                offer.extraData["bookFormat"] = product.extraData.get("bookFormat")
+                offer.extraData["prix_livre"] = product.extraData.get("prix_livre")
+                offer.extraData["editeur"] = product.extraData.get("editeur")
+                offer.extraData["comic_series"] = product.extraData.get("comic_series")
+                offer.extraData["distributeur"] = product.extraData.get("distributeur")
+                offer.extraData["date_parution"] = product.extraData.get("date_parution")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Y a-t-il une raison pour tout expliciter ?
Notamment, est ce qu'on a des valeurs qu'on ne veut pas mettre à jour ? 

Si non : 
```python
offer.extraData.update(product.extraData)
```
Doit suffire

Exemple : 
```python
>>> dict_1 = {"key1": "value", "key2": "value"}
>>> dict_2 = {"key1": "value_1", "key3": "value"}
>>> dict_1.update(dict_2)
>>> print(dict_1)
{'key1': 'value_1', 'key2': 'value', 'key3': 'value'}
```